README.md

```diff
@@ -1 +1,49 @@
-# sale_tracker
\ No newline at end of file
+# Sales Tracker
+
+
+## Cloud Architecture
+
+
+### How it works
+
+
+### Pipeline
+
+
+### Storage
+
+
+### Emailing Service
+
+
+### Dashboard
+
+
+## Entity Relationship Diagram (ERD)
+
+An ERD which clearly describes the tables in the schema and the data stored in each table:
+
+![sale_tracker_ERD](./diagrams/ERD_sale_tracker.png)
+
+### Users Table
+
+This table will be where the users personal information will be uploaded to. This includes their email, first and last name, and phone number. The email and phone number are essential fields that are required depending on whether the user wants to be notified by email or text message. 
+
+### User Preference Table
+
+This table is filled in last as it depends on the product id and user id, whilst inserting the users preference for the price reduction they want to be notified at.
+
```

Comment:
Note that it would be useful to specify individual columns (once we have worked out how to handle deletions).